### PR TITLE
feat: gitcoin v2 logic

### PR DESCRIPTION
**Pull request by nemmtor**

### CODE REVIEW

File: src/application/gitcoin/commands/index.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 
 export const COMMAND_MAP = {
   [GetApplicationsCommand.name]: GetApplicationsCommand,
+  [GetGitcoinAcrossChainFeesCommand.name]: GetGitcoinAcrossChainFeesCommand,
+  [GetNonceCommand.name]: GetNonceCommand,
 } as const;
 
 export { GetApplicationsCommand } from './get-applications';
+export { GetGitcoinAcrossChainFeesCommand } from './get-gitcoin-across-chain-fees';
+export {GetNonceCommand} from './get-nonce'
```
Comment (SolutionsEngineer) [inline src/application/gitcoin/commands/index.ts:13]: small formatting issue here

File: src/application/gitcoin/hooks/use-across-donate-transaction.ts [hunk 2]
```diff
@@ -38,36 +39,50 @@ export const useAcrossDonateTransaction = () => {
       userAmountInWei,
       chainId,
     }: Properties) => {
+      //  ToDo: check where/when chain switch is happening
       const signer = createSigner(wallet);
 
+      console.log('GENERATING VOTE with', signer);
```
Comment (SolutionsEngineer) [inline src/application/gitcoin/hooks/use-across-donate-transaction.ts:45]: Are the console.logs left on purpose?

File: src/shared/idriss/components/send/container.tsx [hunk 3]
```diff
@@ -80,6 +82,7 @@ export const Container = memo(
         event.preventDefault();
         setIsVisible(true);
         updatePosition();
+        onOpen?.();
```
Comment (SolutionsEngineer) [inline src/shared/idriss/components/send/container.tsx:85]: Oh, I've never seen that way of conditional function invocation, and I like it 👌👌

**Review by SolutionsEngineer — APPROVED** [review]

File: src/application/gitcoin/hooks/use-across-donate-transaction.ts (hunk 2 above)
Comment (nemmtor) [reply src/application/gitcoin/hooks/use-across-donate-transaction.ts]: fixed

**Review by lennardevertz — APPROVED** [review]

**Pull request merged**